### ISSUE TITLE
Tablist and Nametags are not formatting correctly.

### ISSUE THREAD

**Issue by Ash10744** [GeyserMC/Geyser#4209]
### Describe the bug

When looking at the tab (In java) it is not organizing the users properly since I installed geyser.

On top of this I am getting a white username underneath every nametag also.

### To Reproduce

Plugin for tablist: 
Bungeetablistplus

Plugin for usernames / nametags:
TAB

### Expected behaviour

It will organize the tablist correctly and the nametags don't have a  additional white name beneath them

### Screenshots / Videos

![image](https://github.com/GeyserMC/Geyser/assets/16086663/ea3f7884-5246-4ffe-a85f-188f4eb636a1)
![image](https://github.com/GeyserMC/Geyser/assets/16086663/c57e4a4b-bcdd-4875-8a47-3d2987f9fe9a)

### Server Version and Plugins

![image](https://github.com/GeyserMC/Geyser/assets/16086663/44a12dc8-e3d8-4354-b694-a48313057b45)
![image](https://github.com/GeyserMC/Geyser/assets/16086663/c8cf17c7-3737-46d6-9131-6ea1fe7f52a1)

### Geyser Dump

https://dump.geysermc.org/n9uWM5Fj1PRpS1xWhLz745Y67xqpw3ld

### Geyser Version

2.2.0-SNAPSHOT

### Minecraft: Bedrock Edition Device/Version

PC

**Comment by Ash10744** [comment]
I have geyser installed on a bungee network scale with floodgate. As I hold multiple servers. So the geyser is fully bungee.

**Comment by onebeastchris** [comment]
Does the second white username only appear after switching servers? 
As for the tab list, that is likely not a Geyser issue - how entries are displayed for java clients isnt something Geyser changes

**Comment by Ash10744** [comment]
Yeah seems so, I managed to fix tablist by switching my plugin to TAB but the nametags still do that

**Comment by Tim203** [comment]
This issue is almost a year old at this point. Please try again with the latest build (685) as that includes a scoreboard rework, including nametags.
